src/Nethermind/Nethermind.Specs/RopstenSpecProvider.cs

```diff
@@ -55,17 +55,23 @@ public IReleaseSpec GetSpec(long blockNumber)
             {
                 return Istanbul.Instance;
             }
+            
```

Comment:
Could we rewrite all those methods using TransitionBlocks ?

Reply:
hmmm, transition blocks were added later, not 100% sure there would be no performance hit
out of the scope of this change